uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/by-unit-and-days-mode/days-unit-multiple-days.html

```diff
@@ -0,0 +1,50 @@
+<ng-include src="$root.sharedPathTpl + 'components/leave-request-popup/details-tab/by-unit-and-days-mode/partials/days-unit-from-date-and-type.html'"></ng-include>
+<div>
```

Comment:
do we need this tag here?

if it's there for wrapping the whole partial it should also wrap the ng-include.

Reply:
There is nothing to wrap here, this is simply a stale old useless div.